src/main.c

```diff
@@ -103,30 +113,66 @@ int load_conf_file(char *file_path)
     return 0;
 }
 
+static void run_test(int test_n) {
+    for (int i = 0; i < tests_exclude.size; i++) {
```

Comment:
Since this can become an exponential-time check, it might be better to implement with a bit set instead of a vector, so that this search loop can be changed into a much faster index-based bit test.

Reply:
integer (4 bytes) can only hold up to 32 bits and long long (8 bytes) can hold up to 64 bits. We have 378 kernel apis which need 6 of long long values. Plus this file is in C language which doesn't add such flexibility to extend such support.

Meaning we would have to write a utility to able support your suggestion. Unless there's something I do not know about? I rather avoid using third-party library nor create our own library for this type of complex.

Reply:
Maybe https://en.cppreference.com/w/cpp/utility/bitset helps

Reply:
Yes, if this file was using C++ language, then I could had used the standard library's bitset class. Except it is not the main objective for this pull request. ergo720 did made a ticket #66 for suggestion change custom library to std::vector. This main.c file could change to C++ language to take advantage of C++ standard library usage.

Plus we also need to be sure that nxdk's custom libcxx can support bitset.

Keep in mind that nxdk's custom libcxx/pbclib are behind from upstream.

Reply:
Transfer to ticket #101